lib/factory_bot/definition_proxy.rb

```diff
@@ -176,6 +176,10 @@ def trait(name, &block)
       @definition.define_trait(Trait.new(name, &block))
     end
 
+    def traits_for_enum(attribute_name, values = nil)
```

Comment:
TODO: Add documentation for this method

Reply:
@composerinteralia Added some documentation, feel free to give me some feedback if you see anything that can be improved!